usaspending_api/awards/v2/views/award_spending.py

```diff
@@ -115,7 +117,7 @@ def get_queryset(self):
 
         # Sum Obligations for each Recipient
         queryset = (
-            queryset.values("award_category", "recipient_id", "recipient_name")
```

Comment:
I don't know where this endpoint is used. I would assume aggregations on DUNS would be preferable over names (unless DUNS is null)